webpack.config.js

```diff
@@ -4,18 +4,39 @@ const WebpackAutoInject = require('webpack-auto-inject-version');
 
 module.exports = (env) => {
   const minimize = env && env.production;
-  const node = env && env.node;
-  const file = minimize ? 'streaming-client.min' : 'streaming-client';
-  const extension = node ? '.cjs' : '.js';
-  const filename = file + extension;
+  const cdn = env && env.cdn;
+
+  let filename = 'streaming-client';
+  let babelExcludes = [];
+
+  /* if building for the cdn */
+  if (cdn) {
+    /*
+      this is so babel doesn't try to polyfill/transpile core-js (which is the polyfill)
+        and the build tools.
+      But we want it polyfill/transpile all other node_modules when building for the web
+    */
+    babelExcludes = [
+      /\bcore-js\b/,
+      /\bwebpack\/buildin\b/
+    ];
+
+    filename += '.bundle';
```

Comment:
I went back and forth on the naming for the cdn build. It was either `streaming-client.bundle.js` or `streaming-client.cdn.js`. Bundle sounded better to me, but I am open to opinions.

Reply:
yeah, i think bundle is better.